cms/static/sass/_base.scss

```diff
@@ -288,14 +288,6 @@ p, ul, ol, dl {
         .button {
           padding: ($baseline/4) ($baseline/2) ($baseline/3) ($baseline/2);
         }
-
-        .new-button {
-
-        }
-
-        .view-button {
-
-        }
```

Comment:
This is good clean up but seems out of scope. Can you move this into its own commit and have a separate PR for it, please.